src/views/iam/TemplateDomainSettings.vue

```diff
@@ -0,0 +1,118 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <div>
+    <a-spin :spinning="currentLoading">
+      <detail-settings
+        :resource="resource"
+        :items="dataSetting"
+        :loading="currentLoading"
+      >
+      </detail-settings>
+    </a-spin>
+  </div>
+</template>
+
+<script>
+import DetailSettings from '@/components/view/DetailSettings'
+import { api } from '@/api'
+
+export default {
+  name: 'TemplateDomainSettings',
```

Comment:
@utchoang @svenvogel this can be removed as we'll have the settings (scoped) implemented via #26

Reply:
I removed this component.